Assets/Scripts/DesignPhilosophy.md

```
@@ -0,0 +1,143 @@
+```csharp
+// UIに渡す情報を持つ。
+public readonly struct ViewModel
+{
+		public ViewModel(int data1, string data2)
+		{
+				Data1 = data1;
+				Data2 = data2;
+		}
+		
+		public readonly int Data1;
+		public readonly string Data2:
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)